yarn-project/end-to-end/src/e2e_blacklist_token_contract/blacklist_token_contract_test.ts

```diff
@@ -222,8 +222,7 @@ export class BlacklistTokenContractTest {
         this.logger.verbose(`Public balance of wallet 0: ${publicBalance}`);
         expect(publicBalance).toEqual(this.tokenSim.balanceOfPublic(address));
 
-        tokenSim.mintPrivate(amount);
-        tokenSim.redeemShield(address, amount);
+        tokenSim.mintPrivate(address, amount);
```

Comment:
tokenSim.redeemShield was misused here and in other places as a way to increase a private balance of someone.